cypress/integration/signup.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Test for Signup', () => {
+  beforeEach(() => {
+    cy.visit('/signup');
+  });
+  it('Image should have alt as signup-icon', () => {
+    cy.get('img')
+      .should('have.attr', 'alt', 'signup-icon');
+  });
+    it('Button should have type submit', () => {
+    cy.get('img')
```

Comment:
This is a repetition of the test above. More so, it should all come under a test case